### ISSUE TITLE
Enhancement - Recently explored clubs

### ISSUE THREAD

**Issue by SnowyNate** [stuysu/stuyactivities.org#737]
When a user goes to the club categories and explores clubs, add a feature that shows recently searched/explored clubs by the user

**Comment by TheEgghead27** [comment]
Also noting that a common request I'd hear is **persisting search page state on navigation** so people can scroll the catalog, click a page to browse, then navigate back to where they left off

(forget if this was mentioned in an issue)

**Comment by SnowyNate** [comment]
Yeah, this issue actually came from a request from a friend of mine
